### PR TITLE
CI: Improve ability to run tests on non-CI clusters

### PR DESCRIPTION
**Pull request by tgraf**
The current tests assume that certain operations are performed in scripts during cluster selection time. This makes it harder to run individual tests on a custom cluster instead of using the standard CI clusters.

This PR achieves two things:
* Deletes all non-essential namespaces (kube-system, default, cilium, ...) before the first test
* Decouples node-labeling from node naming

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
restart-ginkgo

File: test/helpers/kubectl.go [hunk 1]
```diff
@@ -288,18 +288,92 @@ func CreateKubectl(vmName string, log *logrus.Entry) (k *Kubectl) {
...
+		ginkgoext.Failf("Unable to delete non-essential namespaces: %s", err)
+	}
+
+	ginkgoext.By("Labelling nodes")
+	if err = kub.labelNodes(); err != nil {
+		ginkgoext.Failf("unable label nodes: %s", err)
+	}
+}
+
+// labelNodes labels nodes in vagrant env. If you are running tests on
```
Comment (nebril) [inline test/helpers/kubectl.go]: This comment seems to be deprecated by now.

**Review by nebril — CHANGES_REQUESTED** [review]

**Pull request merged**